cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -91,7 +91,12 @@ use bridge_hub_common::{
 use pallet_xcm::EnsureXcm;
 pub use sp_consensus_aura::sr25519::AuthorityId as AuraId;
 pub use sp_runtime::{MultiAddress, Perbill, Permill};
-use xcm::VersionedLocation;
+
+#[allow(unused_imports)]
```

Comment:
why the unused imports?

Reply:
> why the unused imports?

I don't know why `WESTEND_GENESIS_HASH` and `ROCOCO_GENESIS_HASH` are unused variables in this file. Delete them will make bridge not work. To make CI happy, I have to ues `#[allow(unused_imports)]`

Reply:
> will make bridge not work

what do you mean? fails tests? fails compilation? what crate fails tests or compilation?

Reply:
> > will make bridge not work
> 
> what do you mean? fails tests? fails compilation? what crate fails tests or compilation?

For `use xcm::latest::{ROCOCO_GENESIS_HASH, WESTEND_GENESIS_HASH`} in this file, rust analyzer gives warning that `ROCOCO_GENESIS_HASH` and `WESTEND_GENESIS_HASH` are unused variables, so many CI tests like `clippy` failed when push it. But if not import the two constant, compilation failed and report `ROCOCO_GENESIS_HASH` and `WESTEND_GENESIS_HASH` not found. So I have to add `#[allow(unused_imports)]`.

Reply:
what compilation fails? for which crate?

Reply:
> what compilation fails? for which crate?

`bridge-hub-westend-runtime` and `bridge-hub-rococo-runtime`. The detail is here
https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/7075460